code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -713,7 +719,7 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			if(!user.drop_item())
 				return
 			if(U.action(src))
-				to_chat(user, "<span class='notice'>You apply the upgrade to [src].</span>")
+				user.visible_message("<span class = 'notice'>[user] applied the [U] to [src].</span>", "<span class='notice'>You apply the [U] to [src].</span>")
```

Comment:
The [U]?
Or [U]?

Reply:
Should be "the [U]" here, so "Roboticist applied the robotic VTEC Module to Borg"

Reply:
Usually, writing `[U]` like this will append a "the" the front of it, meaning this would print "the the robotic VTEC ...". Did only print one when you tested it?

Reply:
You're right, it shows as "the the".

I tested adding modules at least twice, but I guess my eyes just glazed over it.